Gemfile

```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'rspec'
   gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.9.0')
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '~> 0.11'
+    gem 'sqa-utils', '0.12.1'
```

Comment:
`~> 0.11` in bundler land means "greater than 0.11.0 but less than 1.0.0" which means that puppet-server master and 2.1.x brances will accept 0.13.0 which is between 0.11.0 and 1.0.0.